src/fhost_fpga.py

```diff
@@ -371,8 +407,9 @@ def write_delay(self, source_name, delay=0, delta_delay=0, phase_offset=0,
 
         if delta_phase_offset != 0:
             if act_delta_phase_offset == 0:
-                LOGGER.info('Requested phase offset delta is too slow for this configuration. '
-                    'Setting phase offset change to zero.')
+                LOGGER.info('Requested phase offset delta is too slow for this'
+                            ' configuration.'
+                                'Setting phase offset change to zero.')
```

Comment:
Perhaps merge the third line in the second, or at least wrap it in a more standard way. That's what you get for making minor style changes :)